Description: Use FHS compliant path
 Use FHS compliant path to store the log file by default
 /var/log/siege.log.
Author: Alexandre Anriot <alexandre@atlantilde.com>
Bug-Debian: http://bugs.debian.org/610426
Forwarded: no
Last-Update: 2011-06-30

--- a/configure.in
+++ b/configure.in
@@ -344,7 +344,7 @@
 #define SIEGE_HOME     "$prefix"
 #define URL_FILE       "$sysconfdir/urls.txt"
 #define CNF_FILE       "$sysconfdir/siegerc"
-#define LOG_FILE       "$localstatedir/siege.log"
+#define LOG_FILE       "$localstatedir/log/siege.log"
 
 
 #endif/*JOEPATH_H*/
--- a/doc/siege.1.in
+++ b/doc/siege.1.in
@@ -66,7 +66,7 @@
 TIME, allows you to run the test for a selected period of time.  The format is "NUMm", where NUM is a time unit and the "m" modifier is either S, M, or H for seconds, minutes and hours.  To run siege for an hour, you could select any one of the following combinations: \-t3600S, \-t60M, \-t1H.  The modifier is not case sensitive, but it does require no space between the number and itself.
 .TP
 \fB\-l [FILE]\fR, \fB\-\-log[=FILE]\fR
-LOG transaction stats to FILE. The argument is optional. If FILE is not specified, then siege logs the transaction to SIEGE_HOME/var/siege.log. If siege is installed in /usr/local, then the default siege.log is /usr/local/var/siege.log. This option logs the final statistics reported when siege successfully completes its test. You can edit $HOME/.siegerc to change the location of the siege.log file.
+LOG transaction stats to FILE. The argument is optional. If FILE is not specified, then siege logs the transaction to SIEGE_HOME/var/log/siege.log. This option logs the final statistics reported when siege successfully completes its test. You can edit $HOME/.siegerc to change the location of the siege.log file.
 .TP
 \fB\-m MESSAGE\fR, \fB\-\-mark=MESSAGE\fR
 MARK, mark the log file with a separator.  This option will allow you to separate your log file entries with header information.  This is especially useful when testing two different servers.  It is not necessary to use both the \-m option and the \-l option.  \-m assumes \-l so it marks and logs the transaction. If the MESSAGE has spaces in it, make sure that you put it in quotes.
--- a/doc/siegerc.in
+++ b/doc/siegerc.in
@@ -13,7 +13,7 @@
 # proxy-host = ${PROXY} 
 # You can also reference ENVIRONMENT variables without
 # actually declaring them, example:
-# logfile = $(HOME)/var/siege.log
+# logfile = $(HOME)/var/log/siege.log
 
 #
 # Signify verbose mode, true turns on verbose output
@@ -61,7 +61,7 @@
 logging = true
 
 #
-# Logfile, the default siege logfile is $PREFIX/var/siege.log
+# Logfile, the default siege logfile is $PREFIX/var/log/siege.log
 # This directive allows you to choose an alternative log file.
 # Environment variables may be used as shown in the examples:
 # ex: logfile = /home/jeff/var/log/siege.log
--- a/src/main.c
+++ b/src/main.c
@@ -139,7 +139,7 @@
   puts("  -f, --file=FILE         FILE, select a specific URLS FILE." );
   printf("  -R, --rc=FILE           RC, specify an %src file\n",program_name);
   puts("  -l, --log[=FILE]        LOG to FILE. If FILE is not specified, the");
-  printf("                          default is used: PREFIX/var/%s.log\n", program_name);
+  printf("                          default is used: PREFIX/var/log/%s.log\n", program_name);
   puts("  -m, --mark=\"text\"       MARK, mark the log file with a string." );
   puts("  -d, --delay=NUM         Time DELAY, random delay before each requst");
   puts("                          between 1 and NUM. (NOT COUNTED IN STATS)");
